### DISCUSSION TITLE
Getting data from a few months ago

### DISCUSSION THREAD

**Discussion by bfellerath** [weather-gov/api#281] [Q&A]
I am trying to get data from earlier in the hurricane season (July - August for example) in order to get a real hurricane alert.

When I try to get alerts using 
Start Time:
`2020-07-25T08:00:00+00:00`
End Time:
`2020-08-05T17:00:00+00:00`

in the Response Body I get:
```
{
  "@context": {
    "@version": "1.1"
  },
  "type": "FeatureCollection",
  "features": [],
  "title": "watches, warnings, and advisories issued between Sat, Jul 25, 2020 8:00:00 AM +0000 and Wed, Aug 5, 2020 5:00:00 PM +0000",
  "updated": "2020-09-22T00:00:00+00:00"
}
```

**Comment by JJKraw** [comment]
I've found that the alerts are not preserved long after they expire. For example

  https://api.weather.gov/alerts/NWS-IDP-PROD-4427749-3687428

This is an alert link from September 12. The data is no longer there.
